lib/logging.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+#
+# Logging Library
+# Forked from
+# https://github.com/juan131/bash-libraries/blob/master/lib/liblog.bash
+
+# Color palette
+RESET='\033[0m'
+GREEN='\033[38;5;2m'
+RED='\033[38;5;1m'
+YELLOW='\033[38;5;3m'
+
+source config.sh
+# Log file location (default: "./logger.log")
+LOG_FILE_LOCATION=${LOG_FILE_LOCATION:-"./logger.log"}
+
+# trigger reviewdog
+# Functions
+# also should detect
+$x=1
```

Comment:
🚫 **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don't use $ on the left side of assignments. [SC2281](https://github.com/koalaman/shellcheck/wiki/SC2281)
<!-- __reviewdog__:ChA4NTc5ODE2YTllMWUxMzUyEgpzaGVsbGNoZWNr -->